resources/templates/x86/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "applicationName": "",
+  "buildCommand": "bash /work/compile_app.sh ./src",
+  "buildTarget": "/work/azure-iot-sdk-c/cmake/iot_application"
```

Comment:
Is the project name hard-coded?

Reply:
As I known, 'iot_application' is the default project name which can be changed by user

Reply:
[TODO] Just a placeholder file here. Will modify the file for extensibility later.